src/v2/bandwidthRtc.ts

```diff
@@ -0,0 +1,372 @@
+require("webrtc-adapter");
```

Comment:
I'm the old/current `bandwidthRtc` with some minor import tweaks. Probably don't need to spend a ton of time looking at this one.